src/cephadm/cephadm

```diff
@@ -673,7 +673,7 @@ class CustomContainer(object):
 ##################################
 
 
-def dict_get(d: Dict, key: str, default: Any = None, require: bool = False) -> Any:
+def dict_get(d: Dict, key: str, default: Any = None, require: bool = False) -> Any: # type: ignore
```

Comment:
which type error do you try to avoid here?

Reply:
the 'Any' keywords keep triggering pylance. I know it's a bit selfish to ignore the type errors, but hey, :shrug:

Reply:
the problem is, if you annotate something with `# type: ignore`, you're actually decreasing the testing coverage of the type checker. This is unfortunately more than a simple hint for pylance. 

```python
def fun(i: int) -> "Wrong and should raise an error": # type: ignore
  return i + 1

fun(1)
```
```
(venv) ➜  foo mypy file.py
Success: no issues found in 1 source file
```

Reply:
I fully agree and I don't particularly enjoy the "solution", believe me. I would have loved to have found a proper alternative.

That said, I'm not opposed to reverting those changes, it just seemed the lesser of two evils -- my sanity being lost being the worst of them ;)

Reply:
Hm, we explicitly opted for [allowing `Any` in time signatures](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-disallow-any-decorated). Otherwise we would have set this option in the mypy.ini. 

`# type: ignore` is similar to `reinterpret_cast`: It just allows anything to be done. I wouldn't like to add some `reinterpret_cast`s, just to silence the linter. (Ok, I'm exaggerating here a bit, but you get the idea)

Reply:
ok, I wasn't able to reproduce your linting errors, but after setting the vscode liter to `mypy` (`Ctrl+Shift+P` -> `Select linter` -> `mypy` , everything worked for me after removing those `type: ingore` hints

Reply:
it might have something to do with running the linter in strict mode. I'll try this again later today.